test/dlc_tests/ec2/tensorflow/training/test_tensorflow_training.py

```diff
@@ -159,3 +161,25 @@ def test_tensorflow_addons_cpu(tensorflow_training, ec2_connection, cpu_only):
     if is_tf1(tensorflow_training):
         pytest.skip("This test is for TF2 only")
     execute_ec2_training_test(ec2_connection, tensorflow_training, TF_ADDONS_CMD)
+
+@pytest.mark.parametrize("ec2_instance_type", TF_EC2_CPU_INSTANCE_TYPE, indirect=True)
+def test_tensorflow_dataservice_cpu(tensorflow_training, ec2_connection, cpu_only):
+	ec2_connection.run('python3 -m pip install --upgrade pip')
+	ec2_connection.run('pip3 install tensorflow')
+	ec2_connection.run('pip3 install tf-nightly')
+
+	container_test_local_dir = os.path.join("$HOME", "container_tests")
+	ec2_connection.run(f'cd {container_test_local_dir}/bin && screen -d -m python3 start_dataservice.py')
+
+	execute_ec2_training_test(ec2_connection, tensorflow_training, TF_DATASERVICE_TEST_CMD, host_network=True)
+
+@pytest.mark.parametrize("ec2_instance_type", TF_EC2_GPU_INSTANCE_TYPE, indirect=True)
+def test_tensorflow_dataservice_gpu(tensorflow_training, ec2_connection, gpu_only):
+	ec2_connection.run('python3 -m pip install --upgrade pip')
+	ec2_connection.run('pip3 install tensorflow')
+	ec2_connection.run('pip3 install tf-nightly')
```

Comment:
It is not able to find the package for Tf1.
`E           ERROR: Could not find a version that satisfies the requirement tf-nightly (from versions: none)
E           ERROR: No matching distribution found for tf-nightly`